client/extensions/woocommerce/state/selectors/promotions.js

```diff
@@ -8,3 +8,13 @@ export function getPromotions( rootState, siteId = getSelectedSiteWithFallback(
 	return promotions.promotions;
 }
 
+export function getPromotionsCurrentPage( rootState ) {
+	const { promotions } = rootState.extensions.woocommerce.ui;
+	return promotions.currentPage;
```

Comment:
Would `_.get` be suitable here, used like so?
```js
return get( rootState, 'extensions.woocommerce.ui.promotions.currentPage' )
```

I'm not sure if it's at all possible but for promotions to be `undefined` but `return promotions.currentPage;` would throw if so.

We could also easily add a default as the 3rd argument if that ever became a requirement :)

Reply:
Thanks, @spen. I was on the fence to use get or not. But it's pretty simple here though.
`promotions` should never be undefined because it's a key for `combineReducers`.